sql/core/src/test/resources/sql-tests/inputs/postgreSQL/window_part2.sql

```diff
@@ -15,6 +15,18 @@ CREATE TABLE empsalary (
     enroll_date date
 ) USING parquet;
 
+INSERT INTO empsalary VALUES
```

Comment:
It inserts data into `empsalary` table but why the queries below still show the same results?

Reply:
Because the query statements related to `empsalary` are currently in a comment state. I opened some comments for some of these queries and adjusted the statement and found no query results

Reply:
I see one case being tested against `empsalary`

```sql
select last(salary) over(order by salary range between 1000 preceding and 1000 following),
lag(salary) over(order by salary range between 1000 preceding and 1000 following),
salary from empsalary;
```

```
-- !query
select last(salary) over(order by salary range between 1000 preceding and 1000 following),
lag(salary) over(order by salary range between 1000 preceding and 1000 following),
salary from empsalary
-- !query schema
struct<>
-- !query output
org.apache.spark.sql.AnalysisException
Window Frame specifiedwindowframe(RangeFrame, -1000, 1000) must match the required frame specifiedwindowframe(RowFrame, -1, -1);
```

This isn't supported. Can you comment it out and make a followup JIRA?

Reply:
OK. I created https://issues.apache.org/jira/browse/SPARK-30734 and will update PR